src/server/fmd/slaughter/slaughterman-name/index.test.js

```diff
@@ -0,0 +1,66 @@
+import { describePageSnapshot } from '~/src/server/common/test-helpers/snapshot-page.js'
+import { Answer, slaughtermanNamePage } from './index.js'
+import { SlaughterStubPage } from '../slaughter-stub/index.js'
+
+const sectionKey = 'slaughter'
+const questionKey = 'slaughtermanName'
+const pageUrl = '/fmd/slaughter-information/name'
+const page = slaughtermanNamePage
+const question = 'What is the name of the Slaughterman?'
+
+describe('Answer', () => {
+  it('should have the correct urlPath', () => {
+    expect(page.urlPath).toBe(pageUrl)
+  })
+
+  it('should have the correct sectionKey', () => {
+    expect(page.sectionKey).toBe(sectionKey)
+  })
+
+  it('should have the correct questionKey', () => {
+    expect(page.questionKey).toBe(questionKey)
+  })
+
+  it('should have the correct question', () => {
```

Comment:
there are duplicated tests in this file and we need to add the test for the validation messages